package.json

```diff
@@ -6,6 +6,10 @@
   "homepage": "https://www.powsybl.org/",
   "license": "MPL",
   "repository": "git://github.com/powsybl/powsybl-diagram-viewer.git",
+  "engine": {
```

Comment:
well, it's already like that in commons-ui

Reply:
Yes and if a project import commons-ui it will display the warning

```
npm WARN notsup Unsupported engine for @gridsuite/commons-ui@0.43.0: wanted: {"npm":">=9","node":">=18"} (current: {"node":"14.21.3","npm":"6.14.18"})
npm WARN notsup Not compatible with your version of node/npm: @gridsuite/commons-ui@0.43.0
```

That's why we didn't add `engine` in the projects which import commons-ui

Reply:
Need to remove it except if a TL say no (in this case, we should add it in every project)